### ISSUE TITLE
Seems to not work anymore, nor with atc1441 firmware nor pvvx.

### ISSUE THREAD

**Issue by dglaude** [adafruit/Adafruit_CircuitPython_BLE_LYWSD03MMC#7]
I am testing with multiple NRF board and 7.3.2 such as: `Adafruit CircuitPython 7.3.2 on 2022-07-20; Adafruit Feather Bluefruit Sense with nRF52840`

And I use the following libraries version:
```Found device at /media/dglaude/CIRCUITPY, running CircuitPython 7.3.2.
adafruit_ble_lywsd03mmc==1.0.11
adafruit_ble==8.3.7```

I tested two firmware on two different temperature sensor:
* atc1440: https://github.com/atc1441/ATC_MiThermometer/releases release 78
* pvvx: https://github.com/pvvx/ATC_MiThermometer/raw/master/ATC_v37d.bin version 3.7

In both case the simple test `ble_lywsd03mmc_simpletest.py` only say `Scanning...` in loop, but never connect to the sensor.

Something is broken, I know it has been working in the past as I tested that maybe a year ago, but many variable have changed.

I wonder if someone is using this and what version of CP and library are in use in a working usage.
Also advise on the best firmware+setting to make it work... would be welcome.

But I think the way this library get the information is not power efficient anyway. (more on that later)

**Comment by dhalbert** [comment]
I just fixed a BLE issue that might be this in #6800. Try the artifacts from #6800: https://github.com/adafruit/circuitpython/actions/runs/2915495579

**Comment by dglaude** [comment]
So with those artifact AND after restoring original firmware on the LYWSD03MMC... It is working fine:
```Temperature, Humidity (26.32, 61)
Temperature, Humidity None
Temperature, Humidity (26.3, 61)```

(notice that I sometime have "None", I would say one out of 5 times)

I also reverted from `Adafruit CircuitPython 7.3.2-1-gf03d5892f on 2022-08-23; Adafruit Feather Bluefruit Sense with nRF52840` to `Adafruit CircuitPython 7.3.2 on 2022-07-20; Adafruit Feather Bluefruit Sense with nRF52840` and it still works too.

So the only problem I had was to upgrade to atc1441 and pvvx. Hopefully you reminded me in #8 that this library is supposed to be used with the default firmware...
